Makefile

```diff
@@ -171,3 +171,11 @@ help: ## Prints this help
 		@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) \
 		| sort \
 		| awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: generate-mocks
```

Comment:
how about creating the separated `mockgen.sh` file in the script folder?

Reply:
I've followed the same strategy we are using in the zkevm node to be consistent in all repos